quickstarts/account-data-ingestion/config.yml

```diff
@@ -0,0 +1,37 @@
+# Name of the quickstart (required)
+name: acccount data ingestion
+
+# Displayed in the UI (required)
+title: Data Ingestion Breakdown
+
+# Long-form description of the quickstart (required)
+description: |
+  Shows bytes ingested for APM, Browser, Mobile, Infrastructure, Logs, Synthetics, and Kubernetes.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Shows bytes ingested for different entity types.
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+  - Sri Shetty
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - insights
+  - Data
+  - Ingest
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - teemetry-data-platform
```

Comment:
```suggestion
```
should be `telemetry-data-platform` i assume but...
I don't think this is one of our predefined [install plans](https://github.com/newrelic/newrelic-quickstarts/tree/main/install), is something required to be installed here to push data to the dashboard?

Reply:
Removed this section